### ISSUE TITLE
Cover Art Compatibility with Substreamer

### ISSUE THREAD

**Issue by noah1234j** [kagemomiji/airsonic-advanced#396]
### Scope

GUI

### What problem

Following up on the recent fix for the album art, it was proposed that a new feature request be put in place to try to work on compatibility with substreamer. 

### Feature

Following up on the recent fix for the album art, it was proposed that a new feature request be put in place to try to work on compatibility with substreamer.

**Comment by The-Exterminator** [comment]
Substreamer work fine with the coverart here.

**Comment by noah1234j** [comment]
You are correct, my apologies, it wasn't working for me yesterday. Not sure what changed overnight in my Settup. I had cleared my cache and restarted the server already. Nonetheless it is working now, I'll mark this as complete.
